### PR TITLE
Added Vagrantfile and requirements.txt

### PR DESCRIPTION
**Pull request by gauridhawan**
Added Vagrantfile and requirements.txt file with the required packages.

## Checklist:

- [x] I have attached this PR to an issue/user story

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Flask-API==1.1
+Flask-SQLAlchemy==2.4.1
+python-dotenv==0.10.3
+
+# Testing
+nose==1.3.7
+rednose==1.3.0
+pinocchio==0.4.2
+coverage==4.5.4
+pylint>=2.4.1
```
Comment (iLtc) [inline requirements.txt:12]: This is not a big issue, but I notice that some dependencies have a higher version number now. Should we install the latest one?

This is the latest version I found for some of them:

```
Flask==1.1.2
Flask-API==2.0
Flask-SQLAlchemy==2.4.4
python-dotenv==0.14.0

coverage==5.3
```

Comment (gauridhawan) [reply requirements.txt:12]: I picked up these from the Professor's template project, there might be a compatibility issue?

Comment (iLtc) [reply requirements.txt:12]: I see. Then I think it is fine!

**Review by iLtc — APPROVED** [review]
LGTM. Thanks!

**Review by KHafner — APPROVED** [review]

**Pull request merged**